migrations/20220610154503_delete_package_category_id.sql

```diff
@@ -0,0 +1,3 @@
+alter table package_categories
+drop column if exists package_category_id,
+add constraint package_categories_pkey primary key (package_id, category_id);
```

Comment:
I'd prefer if you edited the migration directly, we don't have to use alter yet.